django_project/frontend/src/containers/MainPage/Trends/index.tsx

```diff
@@ -79,8 +79,8 @@ const Trends = () => {
                     <PopulationTrend
                       selectedSpecies={selectedSpecies}
                       propertyId={propertyId}
-                      startYear={startYear}
-                      endYear={endYear}
+                      startYear={1960}
```

Comment:
Why this specific year?

Reply:
Because that is the minimum allowed year range start.